syntax = "proto3";

message DataPacket {
  enum PacketType {
    TEMPERATURE_C = 0;
    TEMPERATURE_F = 1;
  }

  PacketType packetType = 1;
  int32 value = 2;
}

message EventPacket {
  enum PacketType {
    TIPPER_CLICK = 0;
    SOFT_RESET = 1;
    HARD_RESET = 2;
    START_PAUSE = 3;
    STOP_PAUSE = 4;
  }
  PacketType packetType = 1;
}

